study-builder/studies/osteo/study.conf

```diff
@@ -1343,6 +1347,23 @@
           "expression": "true"
         }
       ]
+    },
+    {
+      "from": {
+        "type": "ACTIVITY",
+        "activityCode": "LOVEDONE"
```

Comment:
Did we mean `FAMILY_HISTORY` here, or are we actually indicating the `LOVEDONE` activity? Are we sure we want to have client go to FAMILY_HISTORY after the LOVEDONE? I don't think this will do anything since no FAMILY_HISTORY instance will exist in the LOVEDONE workflow?